js/index.js

```diff
@@ -0,0 +1,17 @@
+const btn = document.querySelector('#hamburguer-btn');
+const closeBtn = document.querySelector('.close-btn');
+const mobilenavlinks = document.querySelectorAll('.m-navlink');
+
+btn.addEventListener('click', () => {
+  document.getElementById('mobile-menu').style.visibility = 'visible';
+});
+
+closeBtn.addEventListener('click', () => {
+  document.getElementById('mobile-menu').style.visibility = 'hidden';
+});
```

Comment:
- [ ] Kindly make your background match the one in the Figma by making the background blur when clicking on the mobile menu.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/179559559-3090d130-cf3e-4d86-ad4e-3efad7cda91d.png) | ![image](https://user-images.githubusercontent.com/53402600/179550722-1aa655b7-12c0-4f13-ab53-21b874eddb82.png)|

Reply:
Thanks for the feedback! Please check my comment below. Regards.